tests/test_sparkfun_9dof_block.py

```diff
@@ -1,35 +1,36 @@
-from unittest.mock import patch
+import sys
+from unittest.mock import patch, MagicMock
 from collections import defaultdict
 from nio.signal.base import Signal
 from nio.testing.block_test_case import NIOBlockTestCase
-from ..sparkfun_9dof_block import SparkFun9DOF
 from nio.block.terminals import DEFAULT_TERMINAL
 
 
-@patch(SparkFun9DOF.__module__ + '.LSM9DS0')
-class TestiSparkFun9DOF(NIOBlockTestCase):
+class TestSparkFun9DOF(NIOBlockTestCase):
 
     def setUp(self):
         super().setUp()
+        sys.modules['mraa'] = MagicMock()
```

Comment:
Does this seem like a good approach? On OSX or Ubuntu, you may want to run these unit tests even though you can't use `mraa`.